app/ante/evm/eth.go

```diff
@@ -379,31 +382,43 @@ func (issd EthIncrementSenderSequenceDecorator) AnteHandle(ctx sdk.Context, tx s
 			return ctx, errorsmod.Wrap(err, "failed to unpack tx data")
 		}
 
-		// increase sequence of sender
-		acc := issd.ak.GetAccount(ctx, msgEthTx.GetFrom())
-		if acc == nil {
-			return ctx, errorsmod.Wrapf(
-				errortypes.ErrUnknownAddress,
-				"account %s is nil", common.BytesToAddress(msgEthTx.GetFrom().Bytes()),
-			)
-		}
-		nonce := acc.GetSequence()
+		// Sender address should be in the tx cache from the previous AnteHandle call.
+		// From may be either actual from address or the granter address.
+		sender := common.BytesToAddress(msgEthTx.GetFrom())
```

Comment:
so already updated the `msgEthTx.GetFrom` so I guess the entire change for this decorator is not needed?